.gitignore

```diff
@@ -68,3 +68,6 @@ target/
 
 # IntelliJ
 .idea/
+
+# VSCode (We welcome our new Microsoft overlords!)
```

Comment:
Out of curiosity, did you add this yourself, or was it added by Github automatically somehow?

Reply:
I added it. I'm using vs code these days. It's pretty great, actually.